.github/workflows/all.yml

```diff
@@ -55,7 +54,9 @@ jobs:
           go-version: 1.17.x
       - name: Install golangci-lint
         working-directory: /tmp
-        run: go install "github.com/golangci/golangci-lint/cmd/golangci-lint@v$GOLANG_CI_VERSION"
+        run: |
+          GOLANG_CI_VERSION=$(head -n 1 "${GITHUB_WORKSPACE}/.golangci.yml" | sed 's/# //g')
+          go install "github.com/golangci/golangci-lint/cmd/golangci-lint@$GOLANG_CI_VERSION"
```

Comment:
In the event `GOLANG_CI_VERSION` is not also an environment variable pre-defined by the tool itself, what would you think of renaming it to `GOLANGCI_LINT_VERSION` instead? I would find it more unambiguous.